sdk/turing/router/config/router_config.py

```diff
@@ -216,20 +220,40 @@ def to_open_api(self) -> OpenApiModel:
             kwargs['enricher'] = self.enricher.to_open_api()
         if self.ensembler is not None:
             kwargs['ensembler'] = self.ensembler.to_open_api()
+            if kwargs['ensembler'] is None:
+                # Nop ensembler, delete
+                del kwargs['ensembler']
 
         return turing.generated.models.RouterConfig(
             environment_name=self.environment_name,
             name=self.name,
             config=turing.generated.models.RouterVersionConfig(
                 routes=[route.to_open_api() for route in self.routes],
-                default_route_id=self.default_route_id,
```

Comment:
Might be asking something obvious, but why is the `default_route_id` removed from the `RouterVersionConfig`'s init parameters? I was wondering if this would cause any issue with the autogenerated class `RouterVersionConfig` class and on Turing API's end when the OpenAPI specs seem to specify that field as required: https://github.com/krithika369/turing/blob/f8b443975a61d8abaed145970ea2f08aa8d37279/api/api/specs/routers.yaml#L608

Reply:
Oops my bad; I totally forgot you did this here: https://github.com/gojek/turing/pull/192/files/f8b443975a61d8abaed145970ea2f08aa8d37279#diff-49fa6d9d67eac9ae9ac62b762e4db2c2b72de2d407f4a5aa1930e94a0f187a97R213